sphinx_rtd_theme/layout.html

```diff
@@ -26,7 +26,11 @@
     {%- if css|attr("rel") %}
       <link rel="{{ css.rel }}" href="{{ pathto(css.filename, 1) }}" type="text/css"{% if css.title is not none %} title="{{ css.title }}"{% endif %} />
     {%- else %}
-      <link rel="stylesheet" href="{{ pathto(css, 1) }}" type="text/css" />
+      {%- if sphinx_version >= "1.8.0" %}
+        {{ css_tag(css) }}
+      {%- else %}
+        <link rel="stylesheet" href="{{ pathto(css, 1) }}" type="text/css" />
+      {%- endif %}
```

Comment:
this can be simplified:

```
  <link rel="stylesheet" href="{{ pathto('_static/' + style, 1) }}" type="text/css" />
  <link rel="stylesheet" href="{{ pathto('_static/pygments.css', 1) }}" type="text/css" />
  {%- for css in css_files %}
     {%- if css|attr("filename") %}
        {{ css_tag(css) }}
     {%- else %}
         <link rel="stylesheet" href="{{ pathto(css, 1) }}" type="text/css" />
     {%- endif %}
  {%- endfor %}
```